src/main/java/uk/gov/ch/model/update/ManagingOfficerData.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.ch.model.update;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Entity
+public class ManagingOfficerData {
+    @Id
+    @Column(name = "mo_id")
```

Comment:
Maybe an idea to capitalise these values for consistency as thats how its done for the repository query and in other model classes